server/chat-plugins/repeats.ts

```diff
@@ -4,10 +4,13 @@
  * @author Annika, Zarel
  */
 
+import {roomFaqs, getAlias} from './room-faqs';
+
 export interface RepeatedPhrase {
 	phrase: string;
 	/** interval in milliseconds */
 	interval: number;
+	faq?: string;
```

Comment:
I wonder if it would make more sense to give each repeat an ID of some kind that can be used to remove it. That would solve the issue of not being able to remove repeats with commas in them, and then I think you could just set `phrase` to be the FAQ text.

Reply:
Something like ``/repeat [minutes], [name], [phrase]``.

Reply:
(Either of us could also do this in a separate PR; I'm fine with this implementation for now.)

Reply:
Good idea, but I'll leave that to another PR